### PR TITLE
IPA tests pass for new config system

### PR DESCRIPTION
**Pull request by joshdholtz**
@KrauseFx Wooooooooooo :rocket:

### CODE REVIEW

File: lib/fastlane/actions/ipa.rb [hunk 1]
```diff
@@ -74,10 +74,23 @@ def self.run(params)
         ENV[SharedValues::DSYM_OUTPUT_PATH.to_s] = absolute_dsym_path
       end
 
-      def self.params_to_build_args(params)
-        # Remove nil value params unless :clean or :archive
-        params = params.delete_if { |k, v| (k != :clean && k != :archive) && v.nil? }
-
+      def self.params_to_build_args(config)
+        params = {}
+        params[:workspace] = config[:workspace]
```
Comment (KrauseFx) [inline lib/fastlane/actions/ipa.rb]: If you pull from `fastlane_core`, you can now replace all that with
```ruby
params = config.values
```

:rocket:

**Comment by KrauseFx** [comment]
So cool, thank you! :rocket:

File: lib/fastlane/actions/ipa.rb (hunk 1 above)
Comment (joshdholtz) [inline lib/fastlane/actions/ipa.rb]: I tried that earlier but that didn't work so I will try it again :grin:

**Comment by KrauseFx** [comment]
Yes, pull the latest core to get it to work. It was private before.

**Comment by joshdholtz** [comment]
@KrauseFx Updated! The tests failed but I don't think that is because of the IPA stuff, correct?

**Comment by KrauseFx** [comment]
Yes, the tests of the not yet adapted actions fail :disappointed: 

Thanks for the update

**Pull request merged**